base/filesystem.jl

```diff
@@ -219,7 +219,7 @@ function sendfile(dst::File, src::File, src_offset::Int64, bytes::Int)
     check_open(dst)
```

Comment:
While you're at it, Csize_t is unsigned, inconsistent with bytes::Int there and shouldn't it be changed to bytes::UInt? I'm not sure what happens now, cast or maybe corruption if you call with negative values, but if you change will you then at least get a runtime error?